src/modules/place/PlaceView.js

```diff
@@ -0,0 +1,266 @@
+      }
+    }
+    return categories.slice(0, -2);
+  }
+
+  getAddress = () => {
+    var address = '';
+    if (this.props.place.location) {
+      address = (this.props.place.location.address)
+      ? this.props.place.location.address + ' '
+      : '';
+      address += (this.props.place.location.postalCode)
+      ? this.props.place.location.postalCode
+      : '';
+    }
+    return address;
+  }
+
+  getRatingStyles = () => {
+    return (this.props.place.ratingColor)
+    ? ({backgroundColor: '#' + this.props.place.ratingColor})
+    : '';
+  }
+
+  getRating = () => {
+    return (this.props.place.rating)
+      ? (<Image style={styles.gradient}
+              source={require('../../../assets/gradient.png')}>
+              <View style={[styles.ratingView, this.getRatingStyles()]}>
+                <Text style={styles.rating}>{this.props.place.rating}</Text>
+                <Text style={styles.ratingTotal}>/ 10</Text>
+              </View>
+            </Image>)
+      : (<Image style={styles.gradient} source={require('../../../assets/gradient.png')}/>);
+  }
+
+  getHours = () => {
+    return (this.props.place.hours)
+      ? (this.props.place.hours.status)
+      : '';
+  }
+
+  getContact = () => {
+    return (this.props.place.contact)
+      ? (this.props.place.contact.formattedPhone)
+      : '';
+  }
+
+  getLinkURL = () => {
+    var url = (Platform.OS === 'android')
+      ? 'https://maps.google.com?q='
+      : 'http://maps.apple.com/?q=';
+    return url + this.props.place.location.lat + ',' + this.props.place.location.lng;
+  }
+
+  render() {
+    var spinner = this.props.loading
+      ? (<ActivityIndicator style={styles.spinner} size='large' color='white'/>)
+      : (<View/>);
+
+    return (
+      <View style={styles.container}>
+        <ScrollView>
+          {this.getImage()}
+          <View style={styles.cardInfo}>
+            <Text numberOfLines={2} style={styles.title}>
+              {this.props.place.name}
+            </Text>
+            <Text style={[styles.text, {fontWeight: '500'}]}>
+              {this.getCategories()}
+            </Text>
+            <Text style={[styles.text, {fontWeight: '500', marginBottom: 10}]}>
+              {this.getPrice()}
+            </Text>
+            <Text numberOfLines={2} style={styles.text}>
+              {this.getAddress()}
+            </Text>
+            <Text style={styles.text}>
+              {this.getHours()}
+            </Text>
+            <Text style={styles.text}>
+              {this.getContact()}
+            </Text>
+          </View>
+          <View style={styles.buttonContainer}>
+            <Button
+              text='Yeah, take me there!'
+              buttonStyle={theme.buttons.primary}
+              textStyle={theme.fonts.primary}
+              action={() => Linking.openURL(this.getLinkURL())
+                .catch(err => console.error('An error occurred', err))} />
```

Comment:
Generally we conditionally render buttons depending by checking via `Linking.canOpenUrl` (or something similar) if we can actually do it.